glamour_test.go

```diff
@@ -15,9 +13,9 @@ const (
 )
 
 func TestTermRendererWriter(t *testing.T) {
-	r, err := NewTermRenderer("styles/dark.json", ansi.Options{
-		WordWrap: 80,
-	})
+	r, err := NewTermRenderer(
```

Comment:
This actually slightly changes behavior, but I think I'm OK with the change.

For the record: we used to explicitly load the dark style from disk (as opposed to statikFS) and explicitly set the wrapping-width to match the generated testdata. As it matches the default setting and one could argue changing the default breaks backwards-compatibility, it might actually improve the tests.

Reply:
Added explicit word wrap.

Reply:
@twpayne I think you may have misunderstood my comment. I think it's actually beneficial to not explicitly set the word-wrapping here, as the tests would fail if someone tempers with the default setting, which could be considered breaking compatibility with previous releases.

Reply:
Oops. Commit undone.